doc/Comments.xml

```diff
@@ -150,17 +151,31 @@ Example:
 #!  This is my chapter.
 #!  I document my stuff in it.
 ]]></Listing>
+
+Labels of chapters (for referencing purposes) are <C>Chapter_</C><A>name</A>, where
+all spaces in <A>name</A> are replaced by underscores. The <C>@ChapterLabel</C> <A>label</A> command
+can be used to set a custom label. This will set the label of the chapter to <C>Chapter_</C><A>label</A>.
+
+Additionally, the chapter will be stored as <C>_Chapter_</C><A>label</A><C>.xml</C>.
```

Comment:
This is a remark about an internal feature -- and I don't think we explain anywhere right now how AutoDoc works (i.e. that it generates GAPDoc XML files and then feeds them into GAPDoc), do we? So the above sentence is only useful for the handful of experts who know the AutoDoc internals, and confusing to the rest.

So either remove it; or, better (on the long run): add a section somewhere which sketches how AutoDoc works (or perhaps it is there and I forgot about it; didn't even look now). In that case, I'd write something like this: "As described in REF, AutoDoc prouces GAPDoc XML files as an intermediate step. Each chapter is placed into a separate XML file, with the name <C>_Chapter_</C><A>label</A><C>.xml</C>."